### DISCUSSION TITLE
Image Map

### DISCUSSION THREAD

**Discussion by Skiz54** [Zrips/CMI#7275] [Feature Requests]
Creation of a new module on CMI.
The creation of a custom map with an image.

Description:
Load an image (PNG, JPEG, GIF) from a URL onto a map.
Allow images to be saved on the server and reloaded on restart.
Large images cut in several parts! 

Feature Request Similar: https://github.com/Zrips/CMI/discussions/2663

**Comment by mrfloris** [comment]
there's already an images/ folder for holograms, it would be nice if we could use /cmi createmap imagename, to get it to generate.

However, this is quite an extensive feature and a bit out of scope for a plugin like cmi, I can imagine this won't make it in and is perhaps more suited for a stand alone solution. That said, I'm all for it, haha. fingers crossed
